Frontend/rasandbox/src/App.vue

```diff
@@ -5,8 +5,14 @@
       <input type="text" class="SBinput" v-model="inputText" />
       <button @click="submitGame(inputText)" class="SBbutton">Submit</button>
     </div>
-    <div class="gameBox" v-for="game in gamesList" :key="game.name">
-      {{ game.name }}
+    <div class="columns is-desktop is-multiline is-gapless">
```

Comment:
TODO: write a "GameComponent.vue" that is responsible how a Game-"Panel" looks/behaves